Description: Remove remote images from docs.
 This images does not held any information, but breaks user's privacy.
Author: Alexander Gerasiov <gq@debian.org>
Last-Update: 2018-06-17

--- uncrustify-0.67+dfsg1.orig/documentation/htdocs/index.html
+++ uncrustify-0.67+dfsg1/documentation/htdocs/index.html
@@ -16,25 +16,6 @@
 <div class="node2" align="left">
 <h1>Uncrustify</h1>
 <h2>Source Code Beautifier for C, C++, C#, ObjectiveC, D, Java, Pawn and VALA</h2>
-<table>
- <tr>
-  <td valign="top">
-   <script type='text/javascript' language='JavaScript' src='http://www.ohloh.net/projects/4765/badge_js'></script>
-  </td>
-  <td valign="top">
-   <a href="https://travis-ci.org/uncrustify/uncrustify/builds">
-    <img alt="Travis CI Build Status"
-         src="https://travis-ci.org/uncrustify/uncrustify.svg?branch=master"/>
-   </a>
-  </td>
-  <td valign="top">
-   <a href="https://scan.coverity.com/projects/uncrustify">
-    <img alt="Coverity Scan Build Status"
-         src="https://scan.coverity.com/projects/8264/badge.svg"/>
-   </a>
-  </td>
- </tr>
-</table>
 
 <h2>Introduction</h2>
 <p>The goals of this project are simple:
